### PR TITLE
LUN-500: Sponsor Screen

### PR DESCRIPTION
**Pull request by sarahsporck**
This pull request belongs to an issue on our [bugtracker](https://issues.tuerantuer.org/secure/RapidBoard.jspa?rapidView=27&view=detail).
You can find it there by looking for an issue with the key which is mentioned in the title of this pull request.
It starts with the keyword _LUN_.

### CODE REVIEW

**Review by steffenkleinle — APPROVED** [review]
Is this already testable somewhere? Didn't find any supporters in the three dots menu neither with the production nor the dev cms

Only the overlay tested because of that, code looks good

**Comment by sarahsporck** [comment]
> Is this already testable somewhere? Didn't find any supporters in the three dots menu neither with the production nor the dev cms
> 
> Only the overlay tested because of that, code looks good

Currently not testable, as there are not yet any sponsors created in the cms. That's because I don't have access to the lines cms yet :/ 
I asked in the lunes-app channel for an account but didn't get a reply

**Comment by sarahsporck** [comment]
Now testable 👍🏼

**Comment by steffenkleinle** [comment]
Couldn't get opening the links to work: https://reactnative.dev/docs/linking#canopenurl

File: src/routes/home/components/HomeScreenHeader.tsx [hunk 1]
```diff
@@ -41,34 +45,47 @@ type HomeScreenHeaderProps = {
...
-            />
-            <HiddenItem
-              title={getLabels().general.header.settings}
-              onPress={() => navigation.navigate('Settings')}
-              titleStyle={{ fontSize: hp('2%') }}
-              style={{ height: theme.spacingsPlain.xl }}
-            />
-          </OverflowMenu>
+          <MenuIconContainer>
+            <MenuIconWhite testID='menu-icon-white' onPress={() => setOverlayVisible(true)} />
```
Comment (LeandraH) [inline src/routes/home/components/HomeScreenHeader.tsx:87]: The button to press here is tiny, it's really only the three dots of the icon, whereas previously it was a normal, square area around the icon. At least on iOS emulator, it's pretty hard to press the right spot.

**Comment by LeandraH** [comment]
Much better button, thank you! Unfortunately, the screen now flickers when opening the overlay, at least in simulated and real iOS. I've attached a little video:

https://github.com/digitalfabrik/lunes-app/assets/18513943/4fc2ebe9-8357-4ac2-b79f-d67f0f500505

**Comment by sarahsporck** [comment]
> Much better button, thank you! Unfortunately, the screen now flickers when opening the overlay, at least in simulated and real iOS. 

Should work much better now. Although the modal does not span the whole screen now, as it is displayed in the designs. I'll recheck this with toni. Besides that, it should now work like charm :)

File: android/app/src/main/AndroidManifest.xml [hunk 2]
```diff
@@ -29,4 +29,10 @@
         </intent-filter>
       </activity>
     </application>
+      <queries>
+        <intent>
+            <action android:name="android.intent.action.VIEW" />
+            <data android:scheme="https"/>
```
Comment (steffenkleinle) [inline android/app/src/main/AndroidManifest.xml:35]: We don't need mailto/tel at the moment?

**Review by LeandraH — APPROVED** [review]
Very nice, tested on real iPhone

File: android/app/src/main/AndroidManifest.xml (hunk 2 above)
Comment (sarahsporck) [reply android/app/src/main/AndroidManifest.xml:35]: They were already present for ios. I'd need to ask @SteffiStuffel for that. Therefore, I'll keep it as it is rn.

Comment (steffenkleinle) [reply android/app/src/main/AndroidManifest.xml:35]: I am pretty sure that that was bc of contact information in the imprint or something and iOS just required to explicitely request permissions earlier than android. So I think that we probably need it as well. But not 100% sure so lets wait for Steffi.

**Review by steffenkleinle — APPROVED** [review]

**Comment by sarahsporck** [comment]
I had to refactor the overlay to a sidebar, as the design had changed. Do you mind to review again? :)

**Review by steffenkleinle — APPROVED** [review]
Looks good!

File: android/app/src/main/AndroidManifest.xml (hunk 2 above)
Comment (steffenkleinle) [reply android/app/src/main/AndroidManifest.xml:35]: Ping @SteffiStuffel

**Review by LeandraH — APPROVED** [review]
Agree

File: android/app/src/main/AndroidManifest.xml (hunk 2 above)
Comment (SteffiStuffel) [reply android/app/src/main/AndroidManifest.xml:35]: Sorry for the late answer. Yes mail and tel is used in the imprint.

Comment (steffenkleinle) [reply android/app/src/main/AndroidManifest.xml:35]: Those are not pressable atm, created an issue: https://issues.tuerantuer.org/browse/LUN-511

**Pull request merged**